tools/assemblystats/fasta_summary.pl

```diff
@@ -0,0 +1,776 @@
+#!/usr/bin/env perl
+
+#==============================================================================================
+
+# Script to output statistsics and histograms for reads and contigs/isotigs
+
+
+# Outputs include:
+#    Mean, N50, StdDev or reads or contig lengths,
+#    Mean and Modal read or contig lengths.
+#    Number of reads or contigs > 1 kb in length 
+#    Summed contig length (by number of contigs, in sorted order)
+#    Histogram of read or contig lengths,
+#    Graph of sums of read-lengths
+#    File of reads or contigs sorted by read or contig length
+#    Test for mono/di-nucelotide repeats
+#    Randomly selected reads or contigs
+
+
+# Needs gnuplot installed to create the histograms:
+#   On Fedora/Redhat linux: sudo yum install gnuplot 
+#   On Ubuntu/Debian: sudo apt-get install gnuplot
+
+#  Uses a linux pipe to call gnu-plot directly, rather than as a separate shell script.
+
+# Original written by Sujai Kumar, 2008-09-05 University of Edinburgh
+# Modified by Stephen: 29-Apr-2009:
+# Last changed by Stephen: 9-Aug-2010
```

Comment:
Indicate that yourself and mariam have done changes.

Reply:
Is it necessary to indicate the changes here? No one modified this perl script.

Reply:
Then add nothing:)

Reply:
gotcha, thanks!